websites/docs-smoke-test/src/content/components/code-examples.mdx

```diff
@@ -2,56 +2,28 @@
 title: Code Examples
 ---
 
-# Examples With Multiple Languages
+# Multiple languages
+
+**Default theme: dark**
 
 <MultiCodeExample title="Multilanguage Code Samples">
-  <CodeExample path="example.js" highlightLines={[3]} />
+  <CodeExample path="example.js" />
   <CodeExample path="example.java"/>
-  <CodeExample path="example.console" noPromptLines={[3, 4]} />
 </MultiCodeExample>
 
-# Example With One Language
-
-## JavaScript
-
-<CodeExample path="example.js" title="JavaScript Code Sample" />
-
-## Java
-
-<CodeExample path="example.java" />
-
-## Scala
-
-<CodeExample path="example.scala" />
-
-## CSharp
-
-<CodeExample path="example.cs" title="CSharp Code Sample" />
-
-## Curl
+**With theme: light**
 
-<CodeExample path="example.curl" title="Curl Code Sample" />
-
-## GraphQl
-
-<CodeExample path="example.graphql" title="GraphQl Code Sample" />
-
-## JSON
-
-<CodeExample path="example.json" />
-
-## PHP
-
-<CodeExample path="example.php" />
-
-## Shell
+<MultiCodeExample title="Multilanguage Code Samples" useThemeLight>
```

Comment:
Sorry for being slightly pedantic, but I'd be happier with the grammatically correct word order in the author interface. The target group is tech writers, they are nerds about grammar ;-) 

```suggestion
<MultiCodeExample title="Multilanguage Code Samples" useLightTheme>
```

even better: don't introduce imperative words to the DSL with "use" but stick to declarative words like in the other components: 
```suggestion
<MultiCodeExample title="Multilanguage Code Samples" lightTheme>
```

Reply:
Makes sense, thanks. Will rename the prop.

Reply:
Thinking about it, I’m not sure using light/dark terminology is appropriate here and confusing with a possible website theme.

For instance, in case we do have light/dark theme for the entire website, would the light/dark theme for the code blocks still make sense as a name?

Instead maybe we can go with “primary” and “secondary” names. So the secondary version of code block is the “light” prism version.
The prop then would be `secondaryTheme`.

Reply:
Yes, the name could get awkward with a dark theme, but I don't think it's as serious because a) we don't plan a dark theme for now and b) even if it would not be the default. 

Not sure if "secondaryTheme" is transmitting the meaning really good, but we haven't really defined the meaning.    If we go for a semantic flag I would not restrict it to the look / "theme" though and just call it a boolean "secondary". 

I can live with both "light" and "secondary",  let's use "secondary" (also for an easier path to later switch the default if users prefer light as primary).